common/ux-alignment.cpp

```diff
@@ -0,0 +1,114 @@
+#include "ux-alignment.h"
+#include <imgui.h>
+#include <imgui_impl_glfw.h>
+#include <vector>
+#include <memory>
+
+#ifdef _WIN32
+#undef APIENTRY
+#define GLFW_EXPOSE_NATIVE_WIN32
+#define GLFW_EXPOSE_NATIVE_WGL
+#include <GLFW/glfw3native.h>
+
+#include <Windows.h>
+#endif
+
+#define MAGIC 250
+
+bool is_gui_aligned(GLFWwindow *win)
```

Comment:
This function deserves and requires a great comment.
There is no way to guess from the code what you are trying to do and why